packages/pyml/pyml.20171113/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+name: "pyml"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "Thierry Martinez <martinez@nsup.org>"
+homepage: "http://pyml.gforge.inria.fr"
+bug-reports: "http://pyml.gforge.inria.fr/tracker"
+license: "BSD"
+dev-repo: "https://scm.gforge.inria.fr/anonscm/git/pyml/pyml.git"
+build: [make "all" "pymltop" "pymlutop" {utop:installed} "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+depends: ["ocamlfind" {build}]
+depopts: ["utop"]
+version: "20171113"
```

Comment:
I don't think there's a need to include a `version:` line here. However, you should include a minimum OCaml version (and possibly a maximum one). It's very unusual for packages to support OCaml pre 4.02 for example.